packages/account/src/Containers/Account/page-overlay-wrapper.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { useHistory } from 'react-router-dom';
+import UAParser from 'ua-parser-js';
```

Comment:
instead of importing it in multiple packages, can we make it a common function in shared package and reuse it

Reply:
yeah, thought the same.
will try to refactor